index.bs

```diff
@@ -3776,6 +3784,18 @@ To <dfn>locate nodes using CSS</dfn> with given |navigable|, |context nodes|,
 
 </div>
 
+<div algorithm="locate the container element">
+To <dfn>locate the container element</dfn> with given |navigable|:
+
+1. Let |returned nodes| be an empty [=/list=].
+
+1. If |navigable|'s [=navigable/container=] is not null,
```

Comment:
What if the navigable here isn't actually a child of the navigable the command was sent to? That should return an error I assume.